### PR TITLE
Adds page numbers to navigation arrow and to bottom of page

### PR DESCRIPTION
**Pull request by kentsalcedo**
Adds page count to navigation arrows in lap and up browser size, and at the bottom of the page for mobile size signed-off-by: Kent Salcedo <kentsalcedo@webmocha.com>

### CODE REVIEW

File: src/js/components/Article.js [hunk 1]
```diff
@@ -439,19 +440,29 @@ export default class Article extends Component {
...
       if (! this.state.narrow || this.state.atBottom) {
         if (this.state.selectedIndex > 0) {
+          let controlSpacer = (!this.state.narrow) ? (<Paragraph size="large"> &nbsp; </Paragraph>) : null; 
           controls.push(
-            <Button key="previous" ref='previous'
-              plain={true} a11yTitle={a11yTitle.previous}
-              className={`${CONTROL_CLASS_PREFIX}-left`}
-              onClick={this._onPrevious} icon={<PreviousIcon size="large" />} />
+            <Box key="previous" ref='previous' className={`${CONTROL_CLASS_PREFIX}-left`} direction="row"
+            onClick={this._onPrevious} pad={{vertical: "small", horizontal: "medium"}}>
```
Comment (alansouzati) [inline src/js/components/Article.js]: i think there is a spacing issue here `onClick` needs to be shifted to characters to the right

File: src/js/components/Article.js [hunk 2]
```diff
@@ -491,6 +513,11 @@ export default class Article extends Component {
       controls = this._renderControls();
     }
 
+    let pagination;
+    if (this.props.pagination && this.state.atBottom && 'row' === this.props.direction) {
```
Comment (alansouzati) [inline src/js/components/Article.js]: I believe that `this.state.atBottom` is not needed here.

If the page/chapter does not have a scroll, atBottom will never be true

File: src/scss/grommet-core/_objects.article.scss [hunk 3]
```diff
@@ -105,3 +117,16 @@
...
   width: 100%;
 }
+
+.article__pagination {
+  @include media-query(lap-and-up) {
+    display: none;
+  }
+
+  @include media-query(palm) {
+    position: fixed;
```
Comment (alansouzati) [inline src/scss/grommet-core/_objects.article.scss]: I thought we would be rendering at the bottom of the page without positioning fixed.

File: src/js/components/Article.js (hunk 2 above)
Comment (kentsalcedo) [inline src/js/components/Article.js]: @alansouzati I will remove, thanks!

**Comment by kentsalcedo** [comment]
After squashing unwanted commits, I was unable to rebase. Replacing this PR with https://github.com/grommet/grommet/pull/584

File: src/js/components/Article.js (hunk 2 above)
Comment (kentsalcedo) [inline src/js/components/Article.js]: Related to issue https://github.com/grommet/grommet-estories/issues/303. I am working on a fix for this bug, which will deal with this issue at the same time

**Pull request closed without merging**